### PR TITLE
Update services docs

### PR DESCRIPTION
**Pull request by manuel-cid**

### CODE REVIEW

**Review by pacogomezimatia — APPROVED** [review]

**Review by daniel-grana — APPROVED** [review]

File: _docs/15-service.md [hunk 1]
```diff
@@ -193,64 +189,34 @@ export class StarsWarsService extends OntimizeEEService {
 }
 ```
 
-Once your service is created you must decide if it will be used [in the whole application](#use-your-service-in-the-whole-application) or only in [specific components](#use-your-service-in-a-specific-component).
+Once your service is created you cand [override the Ontimize CRUD methods](#override-crud-methods-using-a-third-party-api) and/or [define new methods](#define-your-own-crud-methods). After that you must decide if the service will be used [in the whole application](#use-your-service-in-the-whole-application) or only in [specific components](#use-your-service-in-a-specific-component).
```
Comment (patricia-martinez-imatia) [inline _docs/15-service.md]: did you want to write 'can'?

File: _docs/15-service.md [hunk 2]
```diff
@@ -259,77 +225,19 @@ public advancedQuery(kv?: Object, av?: Array<string>, entity?: string, sqltypes?
...
-```html
-<o-table attr="starships" entity="starships" columns="name;model;manufacturer;starship_class;crew;passengers"
-  visible-columns="name;model;manufacturer;starship_class;passengers"  pageable="yes" quick-filter="no" insert-button="no" fxFlex
-  service-type="starsWars">
-
-  ...
-
-</o-table>
-```
+> **NOTE:** In most cases the third party API won't offer the same response as OntimizeWeb components need so you have to [addapt the response](#adapt-your-service-response).
```
Comment (patricia-martinez-imatia) [inline _docs/15-service.md]: did you want to write 'adapt'?

File: _docs/15-service.md [hunk 3]
```diff
@@ -339,213 +247,158 @@ In the example below we have defined the `getSkywalker` method that retrieves in
...
+    ...
+    { provide: O_DATA_SERVICE, useValue: StarsWarsService }
+  ]
+})
+export class AppModule { }
+```
 
-import { CustomServiceResponse } from './custom-service-response.class'; //Explained on next section
-import { CustomResponse } from './custom.service';
+At this point every **OptimizeWeb** component will use your recently created `StarsWarsService` service for communicating with the backend.
```
Comment (patricia-martinez-imatia) [inline _docs/15-service.md]: did you want to write 'OntimizeWeb'?

File: _docs/15-service.md [hunk 4]
```diff
@@ -339,213 +247,158 @@ In the example below we have defined the `getSkywalker` method that retrieves in
...
-You will have 3 functions in BaseServiceResponse that returns true or false to check the response. This functions are:
+### Define your own successful/unsuccessful request callback methods
 
-- isSuccessful()
-- isFailed()
-- isUnauthorized()
\ No newline at end of file
+**OntimizeWeb** defines a successful and unsuccessful request callbacks for each CRUD method, this methods are called when the service receives the response from the API. You can override this methods in order to modify its behaviour. This methods are: `parseSuccessfulMETHODResponse` and `parseUnsuccessfulMETHODResponse` where `METHOD` is `query`, `advancedQuery`, `update`, `insert` or `delete`.
+
+Both services `OntimizeService` and `OntimizeEEService` also have a generic succesful and unsuccessful request callback which are `parseSuccessfulResponse` and `parseUnsuccessfulResponse`. This callbacks are called from the previous explained CRUD method callbacks so user can chose whether to override a particular or the generic method.
```
Comment (patricia-martinez-imatia) [inline _docs/15-service.md]: did you want to write 'choose'?

**Review by patricia-martinez-imatia — APPROVED** [review]

**Pull request merged**